src/components/add-tag-dialog/add-tag-dialog.js

```
@@ -27,17 +31,46 @@ export class AddTagDialogComponent {
   }
 
   addEventListeners() {
-    this.dialog.listen("MDCDialog:accept", this.handleOk.bind(this));
+    this.dialogButtonConfirm.addEventListener(
+      "click",
+      this.handleOk.bind(this)
+    );
     this.dialog.listen("MDCDialog:cancel", this.handleCancel.bind(this));
+    this.tagNameInput.addEventListener(
```

please do not hide error on focus -- you can do it on `input` event